src/azure-firewall/azext_firewall/custom.py

```diff
@@ -929,4 +972,81 @@ def remove_azure_firewall_policy_filter_rule(cmd, resource_group_name, firewall_
             target_rule_collection.rules.remove(rule)
     return client.create_or_update(resource_group_name, firewall_policy_name,
                                    rule_collection_group_name, rule_collection_group)
+
+
+def _judge(x, y):
+    return x if x else y
```

Comment:
You can use `or` instead of this function.
```python
a = x or y
```

Reply:
good advice